client/state/ui/editor/image-editor/test/reducer.js

```diff
@@ -77,6 +78,14 @@ describe( 'reducer', () => {
 
 			expect( state ).to.be.false;
 		} );
+
+		it( 'should change to false on reset all', () => {
```

Comment:
Oh I suppose the describe block above might make this readable enough. 👍 So feel free to ignore the above comment